3684/0/about_proxy_object_project.rb

```diff
@@ -97,11 +126,11 @@ def test_proxy_counts_method_calls
     assert_equal 1, tv.number_of_times_called(:channel=)
     assert_equal 0, tv.number_of_times_called(:on?)
   end
-
+  # :reek:FeatureEnvy
   def test_proxy_can_record_more_than_just_tv_objects
     proxy = Proxy.new('Code Mash 2009')
 
-    proxy.upcase!
+    proxy.upcase
```

Comment:
looks suspicious. seems like your Proxy class is returning already the `upcased` string. try to remove this line.

http://ruby-doc.org/core-2.6.3/String.html#method-i-upcase

Reply:
i don't understand. Proxy take string as a parameter, then it has method that do .upcase to object. Please, explain me what does it mean "seems like your Proxy class is returning already the upcased string."

Reply:
you are already removed subject with the force-push. you even deleted the file itself... a complete mess.)

you can see here
https://github.com/rubizza-camp/summer-2019/pull/194/files#diff-490d72d8ae5d6be465edccb6a58db9c3R133